deploy/config/config_dev_umccr.toml

```diff
@@ -1,5 +1,5 @@
 elsa_endpoint_authority = "elsa-data.dev.umccr.org"
-cache_location = "elsadatalocaldevtestinfr-tempprivatebucket3b80855-p9qoue91haoy"
+cache_location = "elsadatadevinfrastructur-tempprivatebucket3b80855-8hhal0fyrr7r"
```

Comment:
is there a mechanism whereby this can be picked up from env variables? Because the base Elsa infrastructure has a temp directory for these purposes - and we can surface its name via SSM -> env variables of the lambda

Reply:
i.e. does the TOML have any meta syntax to consider env variables?

Reply:
Yeah that's probably a good idea. I can add that in from an SSM variable. Not sure if the Elsa infrastructure publishes that variable though.